stable/cluster-autoscaler/templates/serviceaccount.yaml

```
@@ -6,3 +6,6 @@ metadata:
 {{ include "cluster-autoscaler.labels" . | indent 4 }}
   name: {{ template "cluster-autoscaler.fullname" . }}
 {{- end -}}
+{{- if .Values.rbac.serviceAccountAnnotations }}
+  annotations: {{ toYaml .Values.rbac.serviceAccountAnnotations | nindent 4 }}
```

It whitespace handled correctly as is, or should it be the following instead?

```suggestion
  annotations:
    {{- toYaml .Values.rbac.serviceAccountAnnotations | nindent 4 }}
```

---

I followed same model that was used on `external-dns`

---

Seems fine, thanks.